src/Network/Pusher.hs

```diff
@@ -111,8 +113,8 @@ import Network.Pusher.Protocol
        (ChannelInfoQuery, ChannelsInfo, ChannelsInfoQuery,
         FullChannelInfo, Users)
 import Network.Pusher.Webhook
-       (Webhooks(..), WebhookEv(..), WebhookPayload(..),
-        parseWebhookPayloadReq)
+       (Webhooks(..), WebhookEv(..), WebhookPayload(..),parseAppKeyHdr,parseAuthSignatureHdr,parseWebhooksBody,verifyWebhooksBody,parseWebhookPayloadReq)
+import qualified Data.ByteString.Char8 as B
```

Comment:
It would be clearer if this was imported as `BC`. I seem to remember this is what this bytestring type is imported as elsewhere.